behaviour/graql/language/mutate.feature

```diff
@@ -0,0 +1,695 @@
+      """
+    Given the integrity is validated
+    Given graql define
+      """
+      define
+      animal sub entity, plays flier;
+      pigeon sub animal;
+      """
+    When get answers of graql query
+      """
+      match $x sub pigeon; get;
+      """
+    Then concept identifiers are
+      |     | check | value  |
+      | PIG | label | pigeon |
+    Then uniquely identify answer concepts
+      | x   |
+      | PIG |
+
+
+  @ignore
+  # TODO: re-enable when attribute ownerships are correctly checked when switching supertypes
+  Scenario: assign new supertype with existing data succeeds if the supertypes have the same attributes
+    Given graql define
+      """
+      define
+      name sub attribute, value string;
+      bird sub entity, has name;
+      pigeon sub bird;
+      """
+    Given the integrity is validated
+    Given graql insert
+      """
+      insert $p isa pigeon;
+      """
+    Given the integrity is validated
+    Given graql define
+      """
+      define
+      animal sub entity, has name;
+      pigeon sub animal;
+      """
+    When get answers of graql query
+      """
+      match $x sub pigeon; get;
+      """
+    Then concept identifiers are
+      |     | check | value  |
+      | PIG | label | pigeon |
+    Then uniquely identify answer concepts
+      | x   |
+      | PIG |
+
+
+  # TODO: write this once 'assign new supertype .. with existing data' succeeds if the supertypes have the same attributes
+  Scenario: assign new supertype throws if existing data has attributes not present on the new supertype
+
+  # TODO: write this once 'assign new supertype .. with existing data' succeeds if the supertypes play the same roles
+  Scenario: assign new supertype throws if existing data plays a role that it can't with the new supertype
+
+  # TODO: write this once 'assign new supertype throws if .. data has attributes not present on the new supertype' is written
+  Scenario: assign new supertype throws if that supertype has a key not present in the existing data (?)
+
+  # TODO: write this once 'define new `sub` on relation type changes its supertype' is written
+  Scenario: assign new super-relation throws if existing data has roleplayers not present on the new supertype (?)
+
+  # TODO: write this once 'define new `sub` on attribute type changes its supertype' passes
+  Scenario: assign new super-attribute throws if it has a different value type to the current one (?)
+
+  # TODO: write this if 'assign new super-attribute throws if it has a different value type ..' turns out to not throw
+  Scenario: assign new super-attribute throws if it has existing data and a different value type to the new supertype (?)
+
+  # TODO: write this once 'define new `sub` on attribute type changes its supertype' passes
+  Scenario: assign new super-attribute throws if new supertype has a regex and existing data doesn't match it (?)
+
+
+  Scenario: define additional 'plays' is visible from all children
```

Comment:
these are a better fit for "Inheritance", what do you think?

Reply:
Added "Inheritance" section only containing these four tests.

Reply:
great :)